test/index.js

```diff
@@ -1,32 +1,98 @@
-const fs = require('fs')
-const path = require('path')
+// ------------------------------------
+// #POSTHTML - CONTENT - TEST
+// ------------------------------------
+
 const test = require('ava')
+
+const { join } = require('path')
+const { readFileSync } = require('fs')
+
 const posthtml = require('posthtml')
-const markdown = require('markdown-it')
-const content = require('..')
-const fixtures = path.join(__dirname, 'fixtures')
+const plugin = require('..')
+
+const fixtures = (file) => readFileSync(join(__dirname, 'fixtures', file))
+
+test('1 - Text', (t) => {
+  const html = fixtures('txt.html')
+  const plugins = [ plugin({ txt: () => 'Lorem' }) ]
+
+  posthtml(plugins)
+    .process(html)
+    .then((result) => result.html)
+    .then((html) => {
+      t.regex(/<p>Lorem<\/p>/, html)
+    })
+})
 
-test('basic', (t) => {
-  const html = fs.readFileSync(path.join(fixtures, 'basic.html'))
-  const plugins = [content({ custom: () => 'foo' })]
+test('2 - Strings ES2015', (t) => {
+  const html = fixtures('es6.html')
+  const text = 'exercitation'
+  const plugins = [ plugin({
+    es6: () => `Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+    eiusmodtempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+    veniam, quis nostrud ${text.toUpperCase()} ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+    velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+    cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+    est laborum.`
```

Comment:
Why such a long test string? Couldn't you accomplish the same goal with a string of like 1/4 this length?